### ISSUE TITLE
Implement Perceptron notebook

### ISSUE THREAD

**Issue by vuhung16au** [vuhung16au/nlp-learning-journey#108]
Implement Perceptron notebook.
Save your response to `examples/perceptron.ipynb`

In this notebook
- Use `iris` dataset
- Demonstrate Perceptron with sklearn
- Demonstrate `SGDClassifier` with params: loss="perceptron",
learning_rate="constant", eta0=1 (the learning rate), and penalty=None (no
regularization).
- Run boths and compare the results 

Refer to the text below for sample code & instruction: 
```
Scikit-Learn provides a Perceptron class that implements a single-TLU network. It
can be used pretty much as you would expect—for example, on the iris dataset (intro‐
duced in Chapter 4):
```
import numpy as np
from sklearn.datasets import load_iris
from sklearn.linear_model import Perceptron
iris = load_iris()
X = iris.data[:, (2, 3)] # petal length, petal width
y = (iris.target == 0).astype(np.int) # Iris setosa?
per_clf = Perceptron()
per_clf.fit(X, y)
y_pred = per_clf.predict([[2, 0.5]])
```


You may have noticed that the Perceptron learning algorithm strongly resembles Sto‐
chastic Gradient Descent. In fact, Scikit-Learn’s Perceptron class is equivalent to
using an SGDClassifier with the following hyperparameters: loss="perceptron",
learning_rate="constant", eta0=1 (the learning rate), and penalty=None (no
regularization).
```
